plugins/uefi-capsule/meson.build

```diff
@@ -1,5 +1,11 @@
 if get_option('plugin_uefi_capsule')
-subdir('efi')
+
+if get_option('efi_binary')
+  efi_binary = dependency ('fwupd-efi', required: false)
```

Comment:
does meson not complain about the space after dependency?

Reply:
It might be safer just to rely on efi_binary for 1.5.x afterall?

Reply:
>does meson not complain about the space after dependency?

I don't think meson cares, but I can certainly fixup the style :)

>It might be safer just to rely on efi_binary for 1.5.x afterall?

Can you elaborate by what you mean?  I didn't add any subproject for 1.5.x, it's entirely a user decision whether or not to use it.  If you think it's better to add subproject to all the older releases like we did for master, that's certainly possible.  I just thought it might be a bit much for a stable release when realistically only some packagers will change this on stable releases.

Reply:
The `dependency ('fwupd-efi` is checking the .pc file, no? Isn't it strictly a runtime requirement, not a build time one?

Reply:
Right, it does check the pc file at build time.  So at build time - if the PC file is there and efi_binary was set to true, don't build the binary.  If it's not there build the binary. 

The runtime check we already have in place, so no concerns whether or not binary is built for 1.5.x (will need to check this on older versions too though when backporting)

Reply:
Ahh, so it's a way of ignoring the efi_binary=true if the binary really is already installed. I don't know if that's cunning or insane.

Reply:
Cunning!